src/main.js

```diff
@@ -166,20 +175,30 @@ function showMainPoster() {
   savedPostersSection.classList.add('hidden');
 };
 
+function userCreatedPoster() {
```

Comment:
At this time, the user created motivational poster loads, but then seems to be hidden. Unsure if this is based on our use of interpolation in this function or if another function is overriding the display of the user created poster. We will review at a later time.